crates/matrix-sdk-common/src/linked_chunk/updates.rs

```diff
@@ -79,6 +79,18 @@ pub enum Update<Item, Gap> {
         items: Vec<Item>,
     },
 
+    /// An item has been replaced in the linked chunk.
+    ///
+    /// The `at` position MUST resolve to the actual position an existing *item*
+    /// (not a gap).
+    ReplaceItem {
```

Comment:
Presumably the code doesn't compile at this commit because `match` statements elsewhere don't include `ReplaceItem`? If so, I would mildly prefer if this commit included `todo` impls to make it compile. This will help with any future bisect we need to do.

Reply:
Yep, will fold a few commits together :+1: